src/components/cqlLibraryLanding/cqlLibraryListActionCenter/compareVersionsAction/CompareVersionsAction.tsx

```diff
@@ -0,0 +1,93 @@
+import React, { useCallback, useEffect, useState } from "react";
+import { IconButton } from "@mui/material";
+import Tooltip from "@mui/material/Tooltip";
+import { CqlLibrary } from "@madie/madie-models";
+import { grey, blue } from "@mui/material/colors";
+
+interface PropTypes {
+  libraries: CqlLibrary[];
+  onClick: () => void;
+}
+
+export const NOTHING_SELECTED =
+  "Select 2 instances within the same library set to compare library versions";
+export const DIFFERENT_LIBRARIES =
+  "Select 2 instances within the same library set to compare library versions";
```

Comment:
don't see any difference in message. I would simply rename the variable and use it for both the cases e.g. INVALID_SELECTION or DEFAULT_MESSAGE or something similar